loadtest/store/memstore/store.go

```diff
@@ -108,3 +110,15 @@ func (s *MemStore) RemoveChannelMember(channelId string, userId string) error {
 	delete(s.channelMembers[channelId], userId)
 	return nil
 }
+
+func (s *MemStore) SetTeamMember(teamId string, teamMember *model.TeamMember) error {
+	if s.teamMembers[teamId] == nil {
+		s.teamMembers[teamId] = map[string]*model.TeamMember{}
+	}
+	s.teamMembers[teamId][teamMember.UserId] = teamMember
+	return nil
+}
+
+func (s *MemStore) TeamMember(teamId, userId string) (*model.TeamMember, error) {
+	return s.teamMembers[teamId][userId], nil
```

Comment:
Let's add a check for existence here. Unless you like what TeamMember("TRY", "THIS") returns :)

Reply:
You know I would hate the result of that :grin: 
But if you approve this PR I promise my next one will be [this](https://mattermost.atlassian.net/browse/MM-20958).

Reply:
Actually it's not even that bad. 
https://play.golang.org/p/So5jYixXNNK

Reply:
This was interesting. I did not expect that. But after some ogling at it, it makes sense. If the key doesn't exist, a map returns the zero value of its value, which here is a nil map again. And again passing a key which does not exist, returns the zero value of `*model.TeamMember` which is a nil pointer. :)

Reply:
Yes, we were all quite baffled by this. I guess what made it confusing, at least for me, was that the zero value for map is `nil` so you would expect that trying to access that would cause a panic. Thankfully it wasn't hard to find an explanation in Golang's blog:  

> A nil map behaves like an empty map when reading, but attempts to write to a nil map will cause a runtime panic

Good to know for next time :muscle: